### DISCUSSION TITLE
Entrega de katas

### DISCUSSION THREAD

**Discussion by AlexanddroPL** [LaunchX-InnovaccionVirtual/CursoIntroPython#82] [Q&A] [answered]
Buen dia, comence a elaborar las katas y se mas o menos lo que tengo que hacer, pero mi duda es como va a ser la dinamica de entrega de las mismas.

**Comment by Dfredude** [comment, marked as answer]
Esta duda ya se respondió varias veces, así como se dijo en el live.

Solo sube las katas a un repositorio público y vas a subir el link a una forma que nos va a mandar Fernanda.

**Reply by AlexanddroPL** [reply]
Gracias, no me terminaba de quedar claro.
